TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -59,57 +60,54 @@ def __init__( self, *args, **kwargs ):
     # # transformations types
     self.transformationTypes = None
     # # directory locations
-    self.directoryLocations = None
-    # # transformation metadata
-    self.transfidmeta = None
```

Comment:
We are using "ProdID" and not "TransformationID" in iLCDirac, so we need this option.